code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -15,16 +16,18 @@
 /obj/item/weapon/reagent_containers/glass/bottle/New()
 	..()
 	if(!icon_state)
```

Comment:
This won't even be possible anymore given that the `icon_state` for the base type is explicitly set.